src/main/java/edu/ie3/util/TimeUtil.java

```diff
@@ -19,6 +24,8 @@
   public static final TimeUtil withDefaults =
       new TimeUtil(ZoneId.of("UTC"), Locale.GERMANY, "yyyy-MM-dd HH:mm:ss");
 
+  public static final double UNDEFINED_TIME = Double.NEGATIVE_INFINITY;
```

Comment:
I'm wondering, if this could impose some trouble some time. Maybe this can be resolved with my comment below.